common/gcp_test.go

```diff
@@ -11,6 +13,19 @@ import (
 	"google.golang.org/api/option"
 )
 
+const fakeCredentialsJson = `{
+	"type": "service_account",
+	"project_id": "fake-project-id",
+	"private_key_id": "123456789",
+	"private_key": "-----BEGIN RSA PRIVATE KEY-----\nMIIBOwIBAAJBAO3U0tUlSLIX06qGqalP+MUScgnmB9scOZ/fZNOykU+QLufdgqDe\nA59CfpytNg/zts8BsRgSTRiLs+6jLhjK6LkCAwEAAQJBALDuGibNROaQyTvcUI2P\n2/8oOMRaZ8++kLP56jV/a5DmwIYt5t5c35/LUWR2GA/7nvQvOJ1XZ6U+uyciOKGg\nJ7ECIQD5k+N8jIMJiobULRLAJgEWQat158sWQ3G23NakdJEWlQIhAPPzjhV+iuJ9\nu4SMOP0BLGgbjWQtna75/cOC916EmLSVAiBrBY7MTti2E7ADdhyPRvy6VYi386Cz\nuFIf7w0f0liRDQIgWD/XOndYjq6lU0HWq8/s3Ix7Da5iyJWu8zdBfXPCOjECIQC1\nusZas9Gcfu4oc3g29c0aQ5IozUTnJhAPjljxj3PmZg==\n-----END RSA PRIVATE KEY-----",
```

Comment:
And is this one really fake?... just to be sure.

Reply:
added the command used to generate the key to reassure that it's a thrown-away key.

Reply:
Include the command for generating the fake/test key to make it clearer :-)

Reply:
GoLang has crypto package in stdlib ;)